src/main/java/nextstep/subway/line/application/SectionService.java

```diff
@@ -28,4 +28,12 @@ public LineResponse addSection(final Long id, final SectionAddRequest sectionAdd
         line.addSection(Section.of(upStation, downStation, sectionAddRequest.getDistance()));
         return LineResponse.from(line);
     }
+
+    @Transactional
+    public void deleteSection(final Long id, final Long stationId) {
```

Comment:
```suggestion
    public void deleteSection(final Long lineId, final Long stationId) {
```

아마 당연히 알고계실테지만 체크 못하신 것 같아서 말씀드립니다 :)
조금 더 명확하게 하면 좋을 것 같아서요 👍